projects/plugins/jetpack/extensions/shared/components/stripe-connect-toolbar-group/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { ToolbarButton, ToolbarGroup } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { flashIcon } from '../../icons';
+import useAutosaveAndRedirect from '../../use-autosave-and-redirect';
+import analytics from '../../../../_inc/client/lib/analytics';
+
+import './style.scss';
+
+export default function StripeConnectToolbarGroup( { blockName, connectUrl, isVisible } ) {
+	const { autosaveAndRedirect } = useAutosaveAndRedirect( connectUrl );
+
+	if ( ! isVisible ) {
+		return null;
+	}
+
+	const handleClick = event => {
+		event.preventDefault();
+		analytics.tracks.recordEvent( 'jetpack_editor_block_stripe_connect_click', {
+			block: blockName,
+		} );
+		autosaveAndRedirect( event );
+	};
+
+	return (
+		<ToolbarGroup>
+			<ToolbarButton
+				className="connect-stripe components-tab-button"
+				icon={ flashIcon }
```

Comment:
This is totally a non-blocking comment, I know this is the icon that we use already but I don't like it:
 * It's very abstract - we're connecting a payment provider, lightning connects the sky and earth
 * It's only used here - it's not used on /earn so it isn't reminding people of other pages they've seen and allowing them to make that connection

If an icon is worthwhile on this button at all, isn't there a stripe logo we can use or something?

Reply:
The Stripe logo is apparently just an S, not particularly explanatory. 😄 

![image](https://user-images.githubusercontent.com/2070010/157861700-2f860001-e4cb-4384-adbf-500d9297f357.png)

Let's keep it as a flash for now. We can ping designers in case they have a service-agnostic "connect" icon that could be used here.